doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -4,16 +4,20 @@
 You can set up an automatic cleanup job in {CENTRAL}.
 
 .Procedure
-. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. In {CENTRAL}, go to *Manage* > *Jobs*.
 . Click *New Job*.
-. Enter a name, due date, time, and the following command into the *Type* text field.
+. Enter a *Business Key*, *Due On*, *Retries*, and the following command into the *Type* field.
 +
 [source]
 ----
 org.jbpm.executor.commands.LogCleanupCommand
 ----
 
-. Click *Add Parameter* if you want to use the <<logcleanupcommand_parameters_{context}, parameters>>, enter a parameter in the key section and enter a parameter value in the value section.
+. If you want to use the <<logcleanupcommand_parameters_{context}, parameters>>:
```

Comment:
Minor rewrite suggestion. Less passive.

To add or modify the <<logcleanupcommand_parameters_{context}, parameters>>, complete the following steps:

Heena, do the closing ">>" belong before the word "parameters?

Reply:
Done